packages/flame/lib/src/components/parallax_component.dart

```diff
@@ -65,7 +65,7 @@ class ParallaxComponent<T extends FlameGame> extends PositionComponent
     double? angle,
     Anchor? anchor,
     int? priority,
-  })  : isFullscreen = size == null ? true : false,
```

Comment:
Hahah what a rookie mistake from my side.

Reply:
it happens, hence the lint!